src/mame/includes/twincobr.h

```diff
@@ -11,6 +11,7 @@
 
 #include "cpu/tms32010/tms32010.h"
 #include "machine/74259.h"
+#include "machine/toaplan_gxl.h"
 #include "video/mc6845.h"
 #include "video/bufsprite.h"
 #include "video/toaplan_scu.h"
```

Comment:
Same here, headers should be organised by module, and alphebetised within a module.